docs/OncoTree-Version-Converter.md

```diff
@@ -0,0 +1,17 @@
+# OncoTree Version Converter Tool
+
+The OncoTree Version Converter tool was developed to facilitate forwards- and backwards-mapping between OncoTree release versions. 
+
+## Running the tool
+
+The OncoTree Version Converter tool can be run with the following command:
+
+```
+python ./scripts/convert_oncotree_version.py --input-file <path/to/input/file> --output-file <path/to/output/file> --source-version <source_oncotree_version> --target-version <target_oncotree_version>
+```
+
+**Options**
+- `-i | --input-file`: This is the input file path. It must contain `ONCOTREE_CODE` in the header and it must contain OncoTree codes corresponding to the `<source_oncotree_version>`.
+- `-o | --output-file`: This is the path to the output file that will be generated. It will contain mapped OncoTree codes from `<source_oncotree_version>`-to-`<target_oncotree_version>`
+- `-s | --source-version`: This is the source OncoTree version. The OncoTree codes in the input file must correspond to this version.
```

Comment:
I would add a short statement telling users how to find out what versions are available.
http://oncotree.mskcc.org/api/versions